apyros/metalog.py

```diff
@@ -57,7 +71,11 @@ def getLog( self, prefix ):
                 ret = line.split()[1].strip()
                 assert ret.startswith("logs/")
                 del self.lines[:i+1]  # cut already passed lines
-                return os.path.dirname( self.filename ) + os.sep + ret[4:]
+                if self.zf is None:
+                    return os.path.dirname( self.filename ) + os.sep + ret[5:]
```

Comment:
Some time ago I've started to use forward slashes for all paths. See above, you assert that the path starts with "logs/" which already hardcodes the forward slash. I suggest to go ahead and just use it internally everywhere. The only place it does not work is when you want to change the PATH environment or some other system specific thing. For reading and writing files from python forward slash is ok.